tslint.json

```diff
@@ -58,10 +58,65 @@
       {
         "enforceBuildableLibDependency": true,
         "allow": [],
-        "depConstraints": [
+        "depConstraints": [{
```

Comment:
Nice, setting up all the project dependency relationships is above and beyond for this task :)